### ISSUE TITLE
Textbox crash

### ISSUE THREAD

**Issue by VaibhavDS19** [synfig/synfig#1932]
**Synfig version & platform**:
(we're more likely to help if you're using latest development builds)
Latest version
Windows and Ubuntu
**Issue description**:
Trying to add textbox causes app to crash Mostly happens when there is already a layer below textbox.

**Comment by ice0** [comment]
Can you check if this problem is reproduced on this build? (Windows)
[Synfig Portable x64](https://ci.appveyor.com/api/buildjobs/57lm1rtg0xgjlaly/artifacts/SynfigStudio-1.5.0-testing-2020-12-19-win64-0aa66.zip)
[Synfig Portable x32](https://ci.appveyor.com/api/buildjobs/df216b2w050m9jgs/artifacts/SynfigStudio-1.5.0-testing-2020-12-19-win32-0aa66.zip)

**Comment by VaibhavDS19** [comment]
I am not using it on Windows now but it doesn't seem to have issues on Ubuntu 20.04

**Comment by ice0** [comment]
Ok, so it Windows-only related problem?

**Comment by VaibhavDS19** [comment]
looks like it

**Comment by rodolforg** [comment]
> Can you check if this problem is reproduced on this build? (Windows)
> [Synfig Portable x64](https://ci.appveyor.com/api/buildjobs/57lm1rtg0xgjlaly/artifacts/SynfigStudio-1.5.0-testing-2020-12-19-win64-0aa66.zip)
> [Synfig Portable x32](https://ci.appveyor.com/api/buildjobs/df216b2w050m9jgs/artifacts/SynfigStudio-1.5.0-testing-2020-12-19-win32-0aa66.zip)


@VaibhavDS19 any news about testing on Windows?

**Comment by VaibhavDS19** [comment]
Now seems to work properly, I reinstalled it and seems to work.

**Comment by rodolforg** [comment]
@VaibhavDS19 on those special portable versions?

> Now seems to work properly, I reinstalled it and seems to work.

In your original post, you said the problem happens on ' Latest version': you mean 1.4.0, right? Does this version work fine now after reinstalation?

**Comment by VaibhavDS19** [comment]
Yes, the above version seems to work without any issues after reinstallation.

**Comment by rodolforg** [comment]
@ice0 , it seems he used the official Synfig 1.4.0 version (not the ones you kindly provided for testing) and everything works fine now (somehow).

So is it safe to close, as it seems to be a bad installation issue?
